dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/entity/FavTask.java

```diff
@@ -34,6 +35,8 @@ public class FavTask {
 
     @TableId(value = "id", type = IdType.AUTO)
     private Long id;
+    //task type name
+    @TableField("task_type")
     private String taskName;
```

Comment:
should we also change entity name from `taskName` to `taskType`?

Reply:
done